yarn.lock

```diff
@@ -1188,9 +1188,9 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
-"@centerforopenscience/ember-osf@https://github.com/CenterForOpenScience/ember-osf.git#0.35.0":
+"@centerforopenscience/ember-osf@https://github.com/CenterForOpenScience/ember-osf.git#develop":
```

Comment:
Should we simply pin to the 0.36.0 tag?

Reply:
Whoops. Not sure what happened here, that should be that. I wonder if the merge with develop wrote over that. I'll fix.